specification/security/_types/Privileges.ts

```diff
@@ -194,6 +198,21 @@ export enum ClusterPrivilege {
   write_fleet_secrets
 }
 
+/**
+ * The subset of cluster level privileges that can be defined for remote clusters.
+ * @availability stack
+ */
+export enum ClusterPrivilegeForRemote {
```

Comment:
Let's rename this to `RemoteClusterPrivilege`. I think that matches the existing schema:

- `ClusterPrivilege` (enum) -> `ClusterPrivileges` (class)
- `RemoteClusterPrivilege` (enum) -> `RemoteClusterPrivileges` (class)

Reply:
will do